FOR REVIEW: Update overview.md (#129)
makerdao/endgame-docs: subdaos/overview.md

# SubDAOs

{% hint style="warning" %}
This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date.
{% endhint %}

> SubDAOs are semi-independent, specialized DAOs that are formed by MakerDAO and designed for protocol alignment. There are different types of subDAOs, and each type has a different role. 

Reviewer: I'd remove the "formed by MakerDAO" line. Maybe "SubDAOs are semi-independent DAOs that are linked to Maker Governance. They are designed to increase innovation and experimentation while limiting risk to Maker Core."

## Overview

> SubDAOs have each their own unique governance token, governance processes, and workforce. They are driven by a self-determined, self-aware *meta* (understood as the non-quantifiable values, aspirations, and sensibilities within a human-operated organization - its ethos). 

Reviewer: I think we can drop the *meta* stuff? Your call

## SubDAO Types

> SubDAOs are divided into three types:

Reviewer: Just wondering where the ProtectorDAOs fit into this new classification? The Allocators explicitly say across "DeFi" - is this meant to include "ReFi" too?

- **[FacilitatorDAOs](facilitator.md)** are tasked with organizing the internal mechanisms of MakerDAO, allocatorDAOs, and miniDAOs.
- **[AllocatorDAOs](allocator.md)** can generate Dai directly from MakerDAO and allocate it as they see fit across the broader DeFi ecosystem. They also manage public-facing entry points to the Maker Ecosystem and are able to spin off miniDAOs.
- **[MiniDAOs](mini.md)** are spun off by allocatorDAOs as outgrowths of an idea or product they are interested in further decentralizing, promoting, or otherwise consolidating. MiniDAOs are experimental offshoots and may be very short-lived.

> A further, higher-level division is drawn between miniDAOs and both facilitatorDAOs and allocatorDAOs, whereby the latter are Major SubDAOs (they are incubated directly by MakerDAO) and the former are Minor SubDAOs (they are incubated by allocatorDAOs) &mdash; i.e., facilitatorDAOs and allocatorDAOs are Major SubDAOs; miniDAOs are Minor SubDAOs.

Reviewer: ```suggestion
FacilitatorDAOs and AllocatorDAOs are incubated directly by MakerDAO and are called Major SubDAOs while MiniDAOs are incubated by AllocatorDAOs and are called Minor SubDAOs.
```

Reviewer: For style purposes, let's capitalize the first word of the SubDAO? e.g FacilitatorDAO and not facilitatorDAO
